dashboard/test/factories/factories.rb

```diff
@@ -170,6 +170,16 @@
           create :single_user_experiment, min_user_id: teacher.id, name: evaluator.pilot_experiment
         end
       end
+
+      # We have some teacher records in our system that do not pass validation because they have
+      # no email address.  Sometimes we want to test against this case because we still want features
+      # to work for these teachers.
```

Comment:
@islemaster What type of features would we want teachers without email addresses to still have access to?

Reply:
In particular, basic account management features.  The particular Honeybadger error I'm addressing is the ability to delete one's own account, which seems important even if no email address is present.  Adding an email address to the account is another feature we'd want to keep in working order.

I think we'd be basically okay if everything else was broken - in fact, we should probably pop a dialog or something prompting these teachers to add an email address.  This error seemed worth fixing sooner rather than later, though.